include/revng/Model/CallEdge.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "revng/Model/FunctionEdge.h"
+#include "revng/Model/FunctionEdgeType.h"
+#include "revng/Model/GeneratedCallEdge.h"
+
+/// A CFG edge to represent function calls (direct, indirect and tail calls)
+class model::CallEdge : public model::generated::CallEdge {
+public:
+  using generated::CallEdge::CallEdge;
+
+public:
+  static bool classof(const FunctionEdge *A) {
+    return FunctionEdgeType::isCall(A->Type);
+  }
```

Comment:
Can you make a brief markdown document describing how to create a new class?

How to include, who you need to inherit from, how to import constructors, what typically you have to write by hand (`verify`, `dump`, `classof`), how inheritance works.

Reply:
Done!